src/assets/src/validation.ts

```diff
@@ -56,6 +56,7 @@ const blankText = 'This field may not be left blank.';
 export const queueNameSchema = string().trim().required(blankText).max(100, createRemainingCharsMessage);
 export const queueDescriptSchema = string().trim().max(1000, createRemainingCharsMessage);
 export const meetingAgendaSchema = string().trim().max(100, createRemainingCharsMessage);
+export const queueLocationSchema = string().trim().required(blankText).max(100, createRemainingCharsMessage);
```

Comment:
@mfldavidson and @jlost, there is currently a bug right now where if you don't enter an in-person meeting location and only select Zoom (or types other than "In Person"), the form won't save (because the location field is required). 
<img width="750" alt="Screen Shot 2021-04-26 at 1 07 58 PM" src="https://user-images.githubusercontent.com/35741256/116124122-e0fbd000-a691-11eb-869d-10d4396d7e12.png">

Making this required only in the situation where "In Person" is enabled may be somewhat tricky (tricky but not impossible). The simplest solution would be to just change the schema so it's not required in any situation, but they may not fit the feature requirements.

Reply:
I think that is a fine first pass. For attendees, can we show "The host(s) have not specified an in-person meeting location" if the field is blank in the template?

Reply:
Thanks, @mfldavidson! @vjcao, I think for the simple solution, the steps would be the following (approximate, we'll have to test):
1) Remove the `required()` part of the schema.
2) Change the attendee-facing language so that it includes the text @mfldavidson provided above if the the location is an empty string.

I don't think you can test the bug-situation, but you should be able to tell whether the field is required or not, and see whether the text you add is showing up in the right situations. Let me know if you need help.